modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -195,7 +186,25 @@ data:
         - sourceLabels: [__name__]
           regex: 'my_metric'
           action: keep
-
+----
++
+.Example of forwarding metrics called `my_metric_1` and `my_metric_2` in `my_namespace` namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        writeRelabelConfigs:
+        - sourceLabels: [__name__,namespace]
+          regex: '(my_metric_1|my_metric_2);my_namespace'
+          action: keep 
 ----
 +
 See the link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config[Prometheus relabel_config documentation] for information about write relabel configuration options.
```

Comment:
The link here is misleading because the configuration syntax isn't the same (e.g. camelCase vs. snake_case options)

Reply:
Therefore, I think it would be better to remove, I saw some threads where this cause confusion and we do not want to have something linked in our docs that does not apply :+1: